src/controllers/Login-Controller.coffee

```diff
@@ -87,6 +91,27 @@ class Login_Controller
   render_Page: (page, view_Model)=>
     @.res.send @.jade_Service.render_Jade_File page, view_Model
 
+  webServiceResponse: (methodName,Token,callback)->
+    options =
+      method: 'post',
+      body: {},
+      json: true,
+      headers: {'Cookie':'Session='+Token}
+      url: @.webServices + '/' + methodName
+    request options, (error, response)=>
+      if error
+        logger?.info ('Could not connect with TM 3.5 server')
+        callback null
+      callback response?.body?.d
+
+  redirectIfPasswordExpired: (token,callback)->
+    @.webServiceResponse "Current_User",token,(userProfile)=>
+      if(userProfile?.PasswordExpired)
+        @.webServiceResponse "GetCurrentUserPasswordExpiryUrl",token,(url)->
+          return callback url
+      else
+        return callback null
```

Comment:
you don't need the ```return```(s) in this case. they are redundant (since coffee will add those returns for you) and due to the use of callbacks we don't want that anyway

The idea is that the returns should only be used in the middle of code (to force method completion) or to represent an explicit value return